aspnetcore/fundamentals/routing/samples/6.0/RoutingSample/Routing/NoZeroesRouteConstraint.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace RoutingSample.Routing
+{
+    // <snippet_Class>
+    public class NoZeroesRouteConstraint : IRouteConstraint
+    {
+        private static readonly Regex _regex = new(
+            @"^[1-9]*$",
+            RegexOptions.CultureInvariant | RegexOptions.IgnoreCase,
+            TimeSpan.FromMilliseconds(100));
+
+        public bool Match(
+            HttpContext? httpContext, IRouter? route, string routeKey,
+            RouteValueDictionary values, RouteDirection routeDirection)
+        {
+            if (!values.TryGetValue(routeKey, out var routeValue))
+                return false;
```

Comment:
Super nit: dotnet/runtime skips braces around single line nesting, but dotnet/aspnetcore is staunchly against it. Could we use braces:

```suggestion
            if (!values.TryGetValue(routeKey, out var routeValue))
            {
                return false;
            }
```

Reply:
Same suggestions elsewhere.

Reply:
Yes, of course. I'll fix it everywhere. Thanks!